jsonrpc/server_test.go

```diff
@@ -105,15 +105,15 @@ func TestHandle(t *testing.T) {
 		{
 			"validation",
 			[]jsonrpc.Parameter{{Name: "param"}},
-			func(v validationStruct) (int, *jsonrpc.Error) {
-				return v.A, nil
+			func(v []validationStruct) (int, *jsonrpc.Error) {
+				return v[0].A, nil
```

Comment:
Why not add this as a separate test case? Surely, we want the previous case to still work, right?

Reply:
because it covers all the cases, including the original one.

Reply:
That is making the assumption about the code under test. 

Please add it as separate cases. I think it is useful to have these cases separately so that in future it is easier to identity which case failed more easily.